BlueprintUI/Sources/View Description/ViewDescription.swift

```diff
@@ -45,9 +45,9 @@ public struct ViewDescription {
     private let _apply: (UIView) -> Void
     private let _contentView: (UIView) -> UIView
     
+    private let _onAppear: TransitionAnimation?
+    private let _onDisappear: TransitionAnimation?
     private let _layoutTransition: LayoutTransition
-    private let _appearingTransition: VisibilityTransition?
-    private let _disappearingTransition: VisibilityTransition?
```

Comment:
Swift style generally avoids leaving off the type name from the variable (eg, UIScreen.main), since the type system guides you, which is why I changed this. Would rather not encode the entire type name in its own var – thoughts?

Reply:
Still noodling on these..

Reply:
I havent come up with anything much better here... I still like these names more than the old ones. Thoughts?

Reply:
AppearanceTransition seems reasonable, but a bit wordy; going with that for now

Reply:
(Though, it also contains a `LayoutTransition`, which isnt necessarily related to appearance.. hm)

Reply:
(Going with `AppearanceTransition`)